Update GUI labels for ArcGIS services (#5611)
qgis/QGIS-Documentation: docs/user_manual/plugins/core_plugins/plugins_metasearch.rst

   Managing Catalog Services

The :guilabel:`Services` tab allows the user to manage all available
catalog services.
MetaSearch provides a default list of Catalog Services, which can be

> added by pressing :guilabel:`Add Default Services` button.

Author: ```suggestion
added by pressing the :guilabel:`Add Default Services` button.
```

#. And modify the :guilabel:`Name` or :guilabel:`URL` values
#. Click :guilabel:`OK`.

To delete a Catalog Service entry, select the entry you would like to
delete and click the :guilabel:`Delete` button.
You will be asked to confirm deleting the entry.

> MetaSearch allows for loading and saving connections to an XML file.

Author: ```suggestion
MetaSearch allows loading and saving connections to an XML file.
```

To load a list of entries:

#. Click the :guilabel:`Load` button. A new window will appear.
#. Click the :guilabel:`Browse` button and navigate to the XML file of
   entries you wish to load.
#. Click :guilabel:`Open`. The list of entries will be displayed.
#. Select the entries you wish to add from the list and click
   :guilabel:`Load`.

Click the :guilabel:`Service Info` button to display information about
the selected Catalog Service such as service identification, service
provider and contact information.
If you would like to view the raw XML response, click the
:guilabel:`GetCapabilities Response` button.

> A separate window will open displaying Capabilities XML.

Author: ```suggestion
A separate window will open displaying the Capabilities XML.
```

   Searching catalog services

The :guilabel:`Search` tab allows the user to query Catalog Services
for data and services, set various search parameters and view results.

The following search parameters are available:

* :guilabel:`Keywords`: free text search keywords;
* :guilabel:`From`: the Catalog Service to perform the query against;

> * **Bounding box**: the spatial area of interest to filter on defined

Author: ```suggestion
* **Bounding box**: the spatial area of interest to filter, defined
```

  by :guilabel:`Xmax`, :guilabel:`Xmin`, :guilabel:`Ymax`, and
  :guilabel:`Ymin`.
  Click :guilabel:`Set  Global` to do a  global search, click

>   :guilabel:`Map Extent` to do a search on the visible area only or

Author: ```suggestion
  :guilabel:`Map Extent` to do a search in the visible area only or
```

>   manually enter custom values as desired.

Author: ```suggestion
  enter values manually.
```

Clicking the :guilabel:`Search` button will search the selected Metadata Catalog.

> Search results are displayed in a list and are sortable by clicking on

Author: ```suggestion
Search results are displayed in a list, and can be sorted by clicking on
```

the column header.
You can navigate through search results with the directional buttons
below the search results.

Select a result and:

> #. click the :guilabel:`View Search Results as XML` button to open a
>    window with the service response in raw XML format.
> #. if the metadata record has an associated bounding box, a footprint
>    of the bounding box will be displayed on the map.
> #. double-clicking the record displays the record metadata with any
>    associated access links.
>    Clicking the links opens the link in the user's web browser.
> #. if the record is a supported web service (WMS/WMTS, WFS, WCS,
>    ArcGIS Map Service, ArcGIS Feature Service, etc.), the
>    :guilabel:`Add Data` button will be enabled for the user to add to
>    QGIS.
>    When clicking this button, MetaSearch will verify if this is a
>    valid OWS.
>    The service will then be added to the appropriate QGIS connection
>    list, and the appropriate connection dialog will then appear.

Author: ```suggestion
* click the :guilabel:`View Search Results as XML` button to open a
  window with the service response in raw XML format.
* if the metadata record has an associated bounding box, a footprint
  of the bounding box will be displayed on the map.
* double-clicking the record displays the record metadata with any
  associated access links.
  Clicking the links opens the link in the user's web browser.
* if the record is a supported web service (WMS/WMTS, WFS, WCS,
  ArcGIS Map Service, ArcGIS Feature Service, etc.), the
  :guilabel:`Add Data` button will be enabled for the user to add to
  QGIS.
  When clicking this button, MetaSearch will verify if this is a
  valid OWS.
  The service will then be added to the appropriate QGIS connection
  list, and the appropriate connection dialog will then appear.
```
